assets/src/css/core-components.css

```diff
@@ -154,3 +154,37 @@
 	position: absolute;
 	top: -2px;
 }
+
+/**
+ * Fixed nav bar on both the settings screen and the wizard.
+ */
+.amp-settings-nav {
+	align-items: center;
+	background: var(--amp-settings-color-background);
+	bottom: 0;
+	box-shadow: 0 -5px 15px rgba(0, 0, 0, 0.05);
+	display: flex;
+	justify-content: flex-end;
+	left: 0;
+	position: fixed;
+	right: 0;
+	z-index: 2;
+}
+
+.amp-settings-nav__inner {
+	align-items: center;
+	display: flex;
+	justify-content: space-between;
+	margin: 0 auto;
+	max-width: 1320px;
+	padding: 15px;
+	width: 100%;
+
+	@media screen and (min-width: 783px) {
+		padding: 30px 90px;
+	}
```

Comment:
Minor point: should files like this eventually get renamed to `*.sass`? This isn't valid CSS so my IDE complains:

![image](https://user-images.githubusercontent.com/134745/94976007-5f134200-04c8-11eb-8265-1020ed4fefcc.png)